docs/developer-docs/gas-cost.md

```diff
@@ -4,15 +4,107 @@
 
-In late November 2022, high-replication application subnets have been made available on the Internet Computer. The first such subnets launched with a replication factor in the order of 30, different sizes may become available in the future. Cycles prices for the new high-replication subnets are scaled linearly based on the number of nodes from the base prices for 13-node subnets. The pricing mechanics for the Bitcoin Mainnet API is slightly different, see the [Bitcoin API documentation](./integrations/bitcoin/bitcoin-how-it-works.md) for details.
+:::info
+The estimates below are simply for demonstrating what different sample architectures may cost. The actual cost of your project will vary based on the exact number of resources and canister calls that are used. Therefore, the estimates below should only be used for gaining an idea of what a project may cost, but should not be used for budgeting exact costs. For exact costs, you can calculate using the chart below for more details: [Details: Cost of compute and storage transactions on the Internet Computer](#details-cost-of-compute-and-storage-transactions-on-the-internet-computer).
+:::
+
+:::info
+These estimates use a 13-node subnet. Costs will be different if deployed on a 34-node subnet. Please refer to the chart below for more details: [Details: Cost of compute and storage transactions on the Internet Computer](#details-cost-of-compute-and-storage-transactions-on-the-internet-computer).
+:::
+
+### Single canister
+
+For a single canister that provides a service or function that is called by other canisters through inter-canister calls the following approximate cycles costs would be incurred:
+
+- Canister creation (one-time fee charged when the canister is deployed): 100_000_000_000 cycles.
+
+- Storing 2GB of data: 254_000 cycles per second; 667_585_152_000 per month.
+
+- Using 0.5% compute resources: 5_000_000 cycles per second; 13_141_440_000_000 cycles per month.
+
+- 1_000 ingress messages received per month with 1_000 bytes per message: 3_200_000_000 cycles. 
+
+**Approximate monthly cost:** 13_812_225_000_000 cycles; 13.812225152 XDR. At the exchange rate of 1.308860, this would cost $18.08 USD.
+
+### Simple static website with a frontend and backend canister 
+
+For a simple static website that uses two canisters (one for the website's backend functionality, one for the frontend assets), that is not called by other canisters or performs HTTPS outcalls, the following approximate cycles costs would be incurred:
+
+- Canister creation for two canisters (one-time fee): 200_000_000_000 cycles.
+
+- Storing 8GB of data (across both canisters): 1_016_000 cycles per second, 2_670_340_600_000 cycles per month.
+
+- Using 2% compute resources: 20_000_000 cycles per second; 52_565_760_000_000 cycles per month.
+
+**Approximate monthly cost:** 55_236_101_000_000 cycles; 55.2361006 XDR. At the exchange rate of 1.308860, this would cost $72.30 USD.
```

Comment:
we ought to revise this a bit for two reasons

1. make sure it's accurate -- $72 a month for a static website seems like a lot, especially considering many platforms offer a free tier for static sites
2. provide more relevant comparisons against platforms like netlify. are we comparing apples to apples?

for example, I like how Tesla's [pricing](https://www.tesla.com/model3/design#overview) for their vehicles subtracts the "savings" from the cost of the vehicle. that takes into account the amount a person would spend on maintenance, gas, oil changes, etc.

could we do something similar here? if it is truly $72 a month, how can we make the comparison 1:1 with the [free tier](https://www.netlify.com/pricing/) limits of e.g. netlify?

Reply:
Revised in the following manner:

- Cut the resources used in this example in half to reflect a more basic/realistic static website project:
      - 2% compute --> 1% compute
      - 8GB storage --> 4GB storage
- Added a note about how Web2 services price things (charging fees for additional features/services not just usage), inspiration taken from the Tesla example provided.